src/khoj/routers/web_client.py

```diff
@@ -133,10 +135,16 @@ def content_config_page(request: Request, content_type: str):
         default_copy = constants.default_config.copy()
         default_content_type = default_copy["content-type"][content_type]  # type: ignore
 
-        default_config = TextContentConfig(
-            compressed_jsonl=default_content_type["compressed-jsonl"],
-            embeddings_file=default_content_type["embeddings-file"],
-        )
+        if content_type == "url":
+            default_config = PageContentConfig(
+                compressed_jsonl=default_content_type["compressed-jsonl"],
+                embeddings_file=default_content_type["embeddings-file"],
+            )
```

Comment:
If the endpoint uses a custom config, it's best to use a separate API for updating/deleting its configuration (see `github_config_page` for example).